worker/worker.py

```diff
@@ -714,7 +722,8 @@ def fetch_and_handle_task(worker_info, password, remote, lock_file, current_stat
         if req["version"] > WORKER_VERSION:
             print("Updating worker version to {}".format(req["version"]))
             update()
-        print("Fetch task...")
+        print("Current time is {} {}".format(datetime.datetime.utcnow(), utcoffset()))
```

Comment:
Can help this?
```
>>> print("utc:\t{}\nlocal:\t{}".format(datetime.datetime.utcnow(), datetime.datetime.now()))
utc:    2021-12-24 12:10:00.723139
local:  2021-12-24 13:10:00.723146
```

Reply:
Yes this is one of the suggestions at SO (for t0 one can take any time). I used another suggestion based on the time module.

Note that most of the code is for converting to the standard stringformat e.g. `+0100`.

Reply:
Note I am using `utcoffset()` only for qualifying the utctime on the terminal (otherwise users might get confused). The timestamps in the logfile just use utc.

Reply:
I guess one can replace `t0.timestamp()` by `0`...

Reply:
Informing the users could be with a `UTC: ` or `(UTC)` addition.

Reply:
I changed it this way, since it shaves off one line of code.

Reply:
@vondele Currently it prints
```
Current time is 2021-12-24 13:16:21.469050 +0100
```
I think this is more or less standard to indicate the utcoffset.

Reply:
this is fine with me.... I'm just worried about any code that deals with timezones, daylight saving, and things like that. Somehow I think having it all UTC will simplify things.

Reply:
> Yes this is one of the suggestions at SO (for t0 one can take any time). I used another suggestion based on the time module.
> 
> Note that most of the code is for converting to the standard stringformat e.g. `+0100`.

Little OT: I'm relying less and less on SO for python, some posts have outdated, though still working, solutions. After starting to understand the language, I'm founding helpful the python documentation. In this case the https://docs.python.org/3/library/datetime.html#datetime.datetime.fromtimestamp referenced also the `time` way.